### ISSUE TITLE
[bug]: Breadcrumbs always return .html suffix

### ISSUE THREAD

**Issue by Jordaneisenburger** [magento/pwa-studio#3029]
Because an empty string "" is a falsy in javascript it will always return .html as the suffix.

**See:**
https://github.com/magento/pwa-studio/blob/0573cfca9e6c6d3b9295b61032c217e537f3530d/packages/peregrine/lib/talons/Breadcrumbs/useBreadcrumbs.js#L47

**We can just change it to:**
```
const categoryUrlSuffix = data?data.category.url_suffix:'.html';
```

**Comment by sirugh** [comment]
Nice find :) Wanna open a PR?

**Comment by Jordaneisenburger** [comment]
Already fixed in 9.0 lol
